githubteacher-tampa.md

```diff
@@ -0,0 +1,6 @@
+
+### Where to Eat
+
+- Sponge Docks
```

Comment:
You can eat at the Sponge Docks!? 😨 

![](http://i.giphy.com/ACX85vE4WDutq.gif)